src/EditorFeatures/CSharpTest/CodeActions/InlineTemporary/InlineTemporaryTests.cs

```diff
@@ -642,7 +640,7 @@ class Program
     static void Main()
     {
         int x = 2;
-        Bar(x < x, (x > (1 + 2)));
+        Bar(x < x, x > 1 + 2);
```

Comment:
new result is better.  there is no parse ambiguity here as there aren't parens around `1+ 2`.